### PR TITLE
linter-formatter

### PR DESCRIPTION
**Pull request by NikitaNikolaitsev**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,24 @@
-def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+from typing import Dict, List
```
Comment (wQuelS) [inline app/main.py]: why have you decided to use `typing` module for such simple type hinting?

File: app/main.py [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+    return {
+        "errors": [format_linter_error(error) for error in errors],
+        "path": file_path,
+        "status": "failed" if errors else "passed"
+    }
+
+
+def format_linter_report(linter_report: Dict[str, List[Dict]]) -> List[Dict]:
+    return [format_single_linter_file(file_path, errors)
+            for file_path, errors in linter_report.items()]
```
Comment (wQuelS) [inline app/main.py]: Your dictionaries and list, are formatted correctly, but they should use one style. 
Please, remain consistent and choose one formatting approach to use everywhere.

**Review by wQuelS — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,13 +1,29 @@
...
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
     # write your code here
     pass
+    return {
+        "errors": [format_linter_error(error) for error in errors],
+        "path": file_path,
+        "status": "failed" if len(errors) else "passed"
```
Comment (DevSpaciX) [inline app/main.py]: You can check if errors exists  more simply -> if errors
Oh and tnot all tests were passed. Fix it pls

**Review by DevSpaciX — APPROVED** [review]

File: app/main.py [hunk 4]
```diff
@@ -1,13 +1,29 @@
 def format_linter_error(error: dict) -> dict:
```
Comment (DevSpaciX) [inline app/main.py:1]: Fix pls failed tests :)

**Review by DevSpaciX — CHANGES_REQUESTED** [review]

**Review by Oleg-s-s — APPROVED** [review]
:+1:

**Pull request closed without merging**